render/jpql/src/main/kotlin/com/linecorp/kotlinjdsl/render/jpql/serializer/impl/JpqlCoalesceSerializer.kt

```diff
@@ -0,0 +1,36 @@
+package com.linecorp.kotlinjdsl.render.jpql.serializer.impl
+
+import com.linecorp.kotlinjdsl.Internal
+import com.linecorp.kotlinjdsl.querymodel.jpql.expression.impl.JpqlCoalesce
+import com.linecorp.kotlinjdsl.render.RenderContext
+import com.linecorp.kotlinjdsl.render.jpql.serializer.JpqlRenderSerializer
+import com.linecorp.kotlinjdsl.render.jpql.serializer.JpqlSerializer
+import com.linecorp.kotlinjdsl.render.jpql.writer.JpqlWriter
+import kotlin.reflect.KClass
+
+@Internal
+internal class JpqlCoalesceSerializer : JpqlSerializer<JpqlCoalesce<*>> {
+    override fun handledType(): KClass<JpqlCoalesce<*>> {
+        return JpqlCoalesce::class
+    }
+
+    override fun serialize(part: JpqlCoalesce<*>, writer: JpqlWriter, context: RenderContext) {
+        val delegate = context.getValue(JpqlRenderSerializer)
+
+        writer.write(" ")
```

Comment:
COALESCE does not need whitespace because it is a single expression.

Reply:
It's not like that. If you actually perform the example query, the query strings will appear attached to each other like SELECTCOALE..

Reply:
I checked again and it seemed to be working normally, so I reflected the review. Thank you.